src/executors/default.yml

```diff
@@ -1,4 +1,4 @@
 description: >
-  Single Docker container with Node 16.16.0 and Cypress dependencies
+  Single Docker container with Node 16.16
```

Comment:
The image will get updated but maybe we remove "16.16" just to avoid confusion later.